tasks/section3.yml

```diff
@@ -586,7 +586,7 @@
       - rule_3.6
 
 - name: "SCORED | 3.6.1 | AUDIT | Ensure iptables is installed"
-  command: /bin/true
+  command: rpm -q iptables
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*